### DISCUSSION TITLE
Question: Problems with autoscaling (maybe missunderstanding of the parameters)

### DISCUSSION THREAD

**Discussion by jmas77** [kubeflow/kfserving#2158] [General]
Good Morning,

I am trying to serve a language model with Kserve and I have noticed that It is not able to autoscale properly. I am really new with this and maybe I am misisng something or I miss understand some parameter:

apiVersion: serving.kserve.io/v1beta1
kind: InferenceService
metadata:
  name: sinodate-model
  namespace: kserve-test
  annotations:
    autoscaling.knative.dev/target: "1"
spec:
  predictor:
      maxReplicas: 6
      containers:
       - name: kserve-container
         image: image_container
         resources:
           requests:
             memory: 2Gi
             cpu: 500m
           limits:
             memory: 4Gi
             cpu: 1600m
i see that there is also a parameter that configures the concurrency of requests in a container but not sure how this affects to the autoscaling.
Thanks so much and sorry for bothering you with the question

**Comment by mdrijwan123** [comment]
Hi any updates on this!!

**Comment by rachitchauhan43** [comment]
@jmas77 : How about asking this question in knative serving channel as this would be make more sense there since kserve uses knative's autoscaling ?
You can join the community here: https://join.slack.com/t/knative/shared_invite/zt-1eua5cxic-~vtgdNpo0A3ZST8K0BkC_A
